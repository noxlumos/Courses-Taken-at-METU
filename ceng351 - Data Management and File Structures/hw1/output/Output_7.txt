
--------------------------------------------------------------
*** Q2 ***
publisher_id	average_page_count
34	214.0
55	408.0

--------------------------------------------------------------
