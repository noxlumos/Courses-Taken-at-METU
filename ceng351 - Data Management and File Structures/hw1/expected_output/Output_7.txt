
--------------------------------------------------------------
*** Q2 ***
publisher_id	average_page_count
34	304.2857
55	408.0

--------------------------------------------------------------
